index.d.ts

```diff
@@ -2,11 +2,11 @@ import { Sprite } from '@pixi/sprite';
 import { TextStyle } from '@pixi/text';
 
 export class HTMLText extends Sprite {
-    constructor(text?:string, style?:TextStyle, canvas?:HTMLCanvasElement);
+    constructor(text?:string, style?:Partial<TextStyle>, canvas?:HTMLCanvasElement);
     readonly canvas:HTMLCanvasElement;
     readonly context:CanvasRenderingContext2D;
     text:string;
-    style:TextStyle;
+    style:Partial<TextStyle>;
```

Comment:
This part doesn't seem right looking at the source, this `style` member is always a complete TextStyle object, but definitely Partial for the constructor.
```suggestion
    style:TextStyle;
```